### PR TITLE
Create an interface for the mutations.

### PR DESCRIPTION
**Pull request by jeffreyhelgers**
The setup for the mutations, an interface that will be implemented and an enum with all the different mutation types.

### CODE REVIEW

File: backEnd/src/main/java/mutation/MutationType.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+	SNP(true), DELETION(true), INSERTION(true), TANDEMDUPLICATION(false), 
+	INTERSPERSEDDUPLICATION(false), INVERSION(false), TRANSLOCATION(false);
+	
+	private boolean between;
+	
+	/**
+	 * Create the mutation type.
+	 * @param between The between.
+	 */
+	MutationType(boolean between) {
```
Comment (MatthijsRijlaarsdam) [inline backEnd/src/main/java/mutation/MutationType.java]: what does between mean?

**Comment by MatthijsRijlaarsdam** [comment]
I think mutationInterface should be an abstract class. Getters in an Interface are a bit bad practice.

**Comment by jeffreyhelgers** [comment]
Between is a boolean that is true when the mutations appears between two strands.

**Comment by MatthijsRijlaarsdam** [comment]
lgtm!

**Pull request merged**